src/AppReducer.js

```diff
@@ -79,23 +77,3 @@ export const AdvisorStore = (state = initialState, action) => {
             return state;
     }
 };
-
-export function entitiesDetailsReducer (ActionTypes) {
```

Comment:
You still need this one, just import `InventoryRuleList` from `'@red-hat-insights/insights-frontend-components` and update your `package.json` to `3.29.5`